package.json

```diff
@@ -7,11 +7,9 @@
     "example": "examples"
   },
   "scripts": {
-    "build": "npm run build:css && npm run build:js",
-    "build:js": "browserify site/js/index.js -o site/assets/bundle.js -t [ babelify --presets [@babel/preset-env] --plugins [babel-plugin-transform-class-properties @babel/plugin-proposal-object-rest-spread] ]",
-    "build:css": "postcss site/css/styles.css -o site/assets/styles.css",
-    "test": "eslint site/js/*.js",
-    "start": "node server.js --watch"
+    "build": "npx snowpack@2.14.0",
```

Comment:
I appreciate the automatic install that comes with this, but I think it would be faster on subsequent builds if you add snowpack as a devDep. 🤷 Just a thought.